Assets/Scripts/UIStates/GamePlayMenuState.cs

```diff
@@ -0,0 +1,40 @@
+using UnityEngine;
+using UnityEngine.Events;
+
+public class GamePlayMenuState : UIBaseState

```

Comment:
```suggestion
public class GameplayMenuState : UIBaseState
```
Gameplay is 1 woord.